.eslintrc.js

```diff
@@ -214,7 +214,7 @@ module.exports = {
 		// this is when Webpack last built the bundle
 		BUILD_TIMESTAMP: true,
 	},
-	plugins: [ 'jest', 'jsx-a11y', 'import' ],
+	plugins: [ 'import' ],
```

Comment:
No need to import `jest` because `plugin:jest/recommended` already imports it. Same for `jsx-a11y`.